src/components/ArticlePreview.jsx

```diff
@@ -1,44 +1,41 @@
 import React from 'react';
-import AuthorList from 'components/AuthorList';
-//import image from '../static/sample-issue/images/articles/palestine-hamilton.jpg';
-
 import { Link } from 'react-router';
 
-export default class ArticlePreview extends React.Component {
-
-  render () {
-    let article = this.props.article;
-    return (
-      <div className="article-preview">
-        {/*
-          Featured image
-          TODO: undo hardcode before release
-        */}
-        <img
-          className="article-preview__featured-image"
-          src={article.featuredImage}
-          alt="featured"
-        />
+// Components
+import AuthorList from 'components/AuthorList';
 
-        {/*
-          Article title with link to article
-        */}
-        <div className="article-preview__content">
-          <Link to={'/issue/' + article.issueId + '/' + article.category + '/' + article.slug}>
-            <h3 className="article-preview__content__title">{article.title}</h3>
-          </Link>
+export default function ArticlePreview(props) {
```

Comment:
I think you should extend BaseComponent and do `export default class ArticlePreview extends BaseComponent`.

BaseComponent includes some utility stuff but most importantly it shortcircuits the render, so if nothing has changed it won't render again. So it leads to higher performance. In general unless there a is a specific reason not to, all components should extend basecomponent